test-avocado/README.md

```diff
@@ -0,0 +1,79 @@
+# Porting tests to Avocado
+
+The test suite in this directory is going to replace the old one as
+soon as we can manage.
+
+The rules for tests are a bit different:
+
+ * One or more machines are dedicated to running all our tests: The
+   machines are initialized specifically for this purpose, and will
+   not be reused for anything else after all tests have been run.
+
+ * A test runs as root directly in one of the machines dedicated to
+   the test run.
+
+ * A test is allowed to drastically modify the machine, as long as
+   that doesn't break other tests.
+
+ * The tests should aim to be independent from each other: Although a
+   test run is deterministic and the tests are always run in the same
+   order, implicit or explicit dependencies between tests should be
+   avoided.  If they can't easily be avoided, they need to be
+   documented, of course.
+
+ * This means that tests should clean up after themselves, and restore
+   the machine into the same configuration as it was before the test.
+   (Of course, a test should not aim for rootkit-level stealthiness:
+   It can and should leave entries in the journal behind, etc.)
+
+More concretely:
+
+ * When a test starts, the cockpit socket is already active, but
+   cockpit itself is not running.  After a test, cockpit will be
+   stopped.
+
+ * A test is a Python file in .../test-avocado modeled after
```

Comment:
is the triple dot notation necessary? After all, the readme is inside this directory.